internal/pkg/service/buffer/store/model/conditions.go

```diff
@@ -0,0 +1,43 @@
+package model
```

Comment:
Idem do modelu pridavat dalsi kod, tak som to v tomto PR rozdelil do viacerych suborov, bez zmeny v kode.